client/packages/common/src/intl/locales/en/common.json

```diff
@@ -1957,5 +1957,6 @@
   "warning.cannot-create-placeholder-units": "There is a total of {{allocatedQuantity}} units available. Unable to allocate all {{requestedQuantity}} units.",
   "warning.caps-lock": "Warning: Caps lock is on",
   "warning.field-not-parsed": "{{field}} not parsed",
-  "warning.nothing-to-supply": "Nothing left to supply!"
+  "warning.nothing-to-supply": "Nothing left to supply!",
+  "tooltip.defined-by-gs1-matrix": "Defined by GS1 datamatrix"
```

Comment:
I'm just wondering if this is a bit techy for end user? Or would they know what this means?

Reply:
Yeah I don't know what a better wording would be. Happy to hear suggestions...